src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTapeCorner.java

```diff
@@ -0,0 +1,338 @@
+
+    /**
+     * Start of the collision box at y.
+     */
+    private static final double BOTTOM_COLLISION = 0.0;
+
+    /**
+     * Start of the collision box at x facing North.
+     */
+    private static final double N_START_COLLISION_X = 0.0;
+
+    /**
+     * End of the collision box facing North.
+     */
+    private static final double N_END_COLLISION_X = 0.5625;
+
+    /**
+     * Start of the collision box at z facing North.
+     */
+    private static final double N_START_COLLISION_Z = 0.0;
+
+    /**
+     * End of the collision box facing North.
+     */
+    private static final double N_END_COLLISION_Z = 0.5625;
+
+    /**
+     * Start of the collision box at x facing West.
+     */
+    private static final double W_START_COLLISION_X = 0.0;
+
+    /**
+     * End of the collision box facing West.
+     */
+    private static final double W_END_COLLISION_X = 0.5625;
+
+    /**
+     * Start of the collision box at z facing West.
+     */
+    private static final double W_START_COLLISION_Z = 0.4375;
+
+    /**
+     * End of the collision box facing West.
+     */
+    private static final double W_END_COLLISION_Z = 1.0;
+
+    /**
+     * Start of the collision box at x facing South.
+     */
+    private static final double S_START_COLLISION_X = 0.4375;
+
+    /**
+     * End of the collision box facing South.
+     */
+    private static final double S_END_COLLISION_X = 1.0;
+
+    /**
+     * Start of the collision box at z facing South.
+     */
+    private static final double S_START_COLLISION_Z = 0.4375;
+
+    /**
+     * End of the collision box facing South.
+     */
+    private static final double S_END_COLLISION_Z = 1.0;
+
+    /**
+     * Start of the collision box at x facing East.
+     */
+    private static final double E_START_COLLISION_X = 0.4375;
+
+    /**
+     * End of the collision box facing East.
+     */
+    private static final double E_END_COLLISION_X = 1.0;
+
+    /**
+     * Start of the collision box at z facing East.
+     */
+    private static final double E_START_COLLISION_Z = 0.0;
+
+    /**
+     * End of the collision box facing East.
+     */
+    private static final double E_END_COLLISION_Z = 0.5625;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 1.0;
+    /**
+     * How much light goes through the block.
+     */
+    private static final int LIGHT_OPACITY = 0;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public BlockConstructionTapeCorner()
+    {
+        super(Material.VINE);
+        initBlock();
+    }
+
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(), BLOCK_NAME));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1449)